osu.Game/Rulesets/Edit/Types/IHasEditablePosition.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+
+namespace osu.Game.Rulesets.Edit.Types
+{
+    public interface IHasEditablePosition

```

Comment:
this should probably derive from `IHasPosition` and maybe expose `Postiion`'s `Set`

Reply:
I didn't want to expose `Position`'s setter because it's super hard to control the logic for slider circles - consider that `Position` is set when loading the beatmap...